desktop-widgets/divelistview.cpp

```diff
@@ -407,13 +401,6 @@ void DiveListView::headerClicked(int i)
 
 void DiveListView::reload(DiveTripModel::Layout layout, bool forceSort)
 {
-	// we want to run setupUi() once we actually are displaying something
-	// in the widget
-	static bool first = true;
-	if (first && dive_table.nr > 0) {
-		setupUi();
-		first = false;
-	}
```

Comment:
this was a critical fix way back when it was introduced. Because column widths that you set when nothing was displayed weren't kept.
My guess is that this is a Qt bug that was fixed?